src/main/msp/msp.c

```diff
@@ -3854,6 +3860,13 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
             }
         }
         break;
+    case MSP_SET_LED_STRIP_CONFIG_VALUES:
+        {
+            ledStripConfigMutable()->ledstrip_brightness = sbufReadU8(src);
+            ledStripConfigMutable()->ledstrip_rainbow_delta = sbufReadU8(src);
+            ledStripConfigMutable()->ledstrip_rainbow_freq = sbufReadU8(src);
+        }
+        break;
```

Comment:
```suggestion
    case MSP_SET_LED_STRIP_CONFIG_VALUES:
        ledStripConfigMutable()->ledstrip_brightness = sbufReadU8(src);
        ledStripConfigMutable()->ledstrip_rainbow_delta = sbufReadU8(src);
        ledStripConfigMutable()->ledstrip_rainbow_freq = sbufReadU8(src);

        break;
```

Reply:
I wasn't sure which one to use since it's just reading, will try "Mutable" tomorrow.